Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.yml

```diff
@@ -0,0 +1,2407 @@
+    - description: The maximum number of records to retrieve per page.
+      isArray: false
+      name: page_size
+      required: false
+    - defaultValue: '50'
+      description: The maximum number of records to retrieve.
+      isArray: false
+      name: limit
+      required: false
+    deprecated: false
+    description: List the Certificate intermediate groups.
+    execution: false
+    name: fortiwebvm-certificate-intermediate-group-list
+    outputs:
+    - contextPath: FortiwebVM.CertificateIntermediateGroup.id
+      description: Certificate Intermediate Group Name.
+      type: String
+  - arguments:
+    - description: The page number of the results to retrieve.
+      isArray: false
+      name: page
+      required: false
+    - description: The maximum number of records to retrieve per page.
+      isArray: false
+      name: page_size
+      required: false
+    - defaultValue: '50'
+      description: The maximum number of records to retrieve.
+      isArray: false
+      name: limit
+      required: false
+    deprecated: false
+    description: List the Server Pools.
+    execution: false
+    name: fortiwebvm-server-pool-list
+    outputs:
+    - contextPath: FortiwebVM.ServerPool.id
+      description: Server Pool Name.
+      type: String
+  - arguments:
+    - description: The page number of the results to retrieve.
+      isArray: false
+      name: page
+      required: false
+    - description: The maximum number of records to retrieve per page.
+      isArray: false
+      name: page_size
+      required: false
+    - defaultValue: '50'
+      description: The maximum number of records to retrieve.
+      isArray: false
+      name: limit
+      required: false
+    deprecated: false
+    description: "List the  HTTP Services."
+    execution: false
+    name: fortiwebvm-http-service-list
+    outputs:
+    - contextPath: FortiwebVM.HttpServiceList.id
+      description: HTTP Service Name.
+      type: String
+  - arguments:
+    - description: The page number of the results to retrieve.
+      isArray: false
+      name: page
+      required: false
+    - description: The maximum number of records to retrieve per page.
+      isArray: false
+      name: page_size
+      required: false
+    - defaultValue: '50'
+      description: The maximum number of records to retrieve.
+      isArray: false
+      name: limit
+      required: false
+    deprecated: false
+    description: List the Inline Protection Profiles.
+    execution: false
+    name: fortiwebvm-inline-protection-profile-list
+    outputs:
+    - contextPath: FortiwebVM.InlineProtectionProfile.id
+      description: Inline protection profile name.
+      type: String
+  - arguments:
+    - description: Policy Name.
+      isArray: false
+      name: name
+      required: true
+    - description: Allows the use of the server policy JSON template. All of the arguments listed below will be overridden. For an example, see the integration .
```

Comment:
The last sentence is missing something...
For an example, see the integration ....?